feat: useDeleteMany example for Material UI (#1970)
pankod/refine: documentation/docs/examples/table/mui/useDeleteMany.md

---
id: useDeleteMany
title: useDeleteMany
---

> The `useDeleteMany` is one of **refine**'s data hooks. It removes more than one data from the database. We used the `useDeleteMany` hook to remove more than one record from our table in this scenario. On your table, you may remove multiple records by selecting them and pressing delete. You can get more information by examining the example.

Reviewer: ```suggestion
The `useDeleteMany` is one of the **refine**'s data hooks. It allows you to delete multiple data from your database.  We used `useDeleteMany` to remove multiple records from the table in this scenario. On your table, you may remove multiple records by selecting them and pressing delete. You can get more information by examining the example.
```

Reviewer: I think It would be better use the "multiple" keyword instead of "more that one"
